<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointAndLine</class>
 <widget class="QMainWindow" name="PointAndLine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnProcess">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Process</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>26</y>
      <width>471</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>44</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbLine">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Line</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="cbLineDirection">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>79</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Point</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>228</x>
      <y>4</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Sharpness</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
